### DISCUSSION TITLE
have you seen this man

### DISCUSSION THREAD

**Discussion by HbmMods** [HbmMods/Hbm-s-Nuclear-Tech-GIT#945] [Announcements]
![image](https://user-images.githubusercontent.com/16674258/223833991-7fde70e7-3645-417d-8430-40e72ce063a9.png)

**Comment by Damielle3141** [comment]
Yeah, he's at 53 street with the rundown house in meowcity in kingdom of cats

**Reply by HollyEreTic** [reply]
Oh, I'm your fan, Hbm!
